modules/util/path.go

```diff
@@ -91,7 +90,7 @@ func statDir(dirPath, recPath string, includeDir, isDirOnly, followSymlinks bool
 
 	statList := make([]string, 0)
 	for _, fi := range fis {
-		if strings.Contains(fi.Name(), ".DS_Store") {
+		if fi.IsDir() && CommonSkipDir(fi.Name()) {
```

Comment:
Is `.DS_Store` a directory?

```
$ ls -l .DS_Store
-rw-r--r-- 1 user staff 12292 Aug 27 10:25 .DS_Store
```

Reply:
Sigh... then the original code here was wrong. 

Is there anything else that MacOSX will helpfully add to the directories that we should ignore?

Reply:
Have they perhaps changed this at some point?
Because I'm fairly certain I've already seen the `.DS_STORE` directories in zips from Macs…
(And in the implementation now we don't seem to skip this file anymore?)

Reply:
macOS stores some information in `._*`  files for some filesystems. ref: https://apple.stackexchange.com/questions/14980/why-are-dot-underscore-files-created-and-how-can-i-avoid-them . IMO it doesn't matter in most cases.

And since in most cases the dot-files (`.*`) are for special purposes, so maybe a simple approach is making the watcher ignore all the dot-files. Current approach is also fine. Either works.

Reply:
Yep, just noticed I overlooked the `if file[0] = "."` part…